docs/technical_reference/unit_models/gac.rst

```diff
@@ -314,7 +322,7 @@ Code Documentation
 -------------------
 
 * :mod:`watertap.unit_models.gac`
-* :meth:`watertap.costing.watertap_costing_package.WaterTAPCostingData.cost_gac`
+* :meth:`watertap.costing.watertap_costing_package.units.gac`
```

Comment:
You're probably aware, but this link to the costing method isn't clickable -probably because we don't have documentation for the costing package yet. That's fine for now- just pointing it out just in case you weren't aware.

Reply:
fixed in #960 when I noticed it ☺, just waiting for this or that to merge so I can fix the conflicts in the doc file